server/protocols/jmap/pom.xml

```diff
@@ -66,6 +66,12 @@
             <groupId>javax.annotation</groupId>
             <artifactId>javax.annotation-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+            <version>4.4.13</version>
```

Comment:
please reuse the httpclient version property

Reply:
the version used with that property is `4.5.9` for [httpclient-osgi](https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient-osgi) and [fluent-hc](https://mvnrepository.com/artifact/org.apache.httpcomponents/fluent-hc) in our corde.

However httpcore seems to follow an other versioning, as this one doesn't exist: https://mvnrepository.com/artifact/org.apache.httpcomponents/httpcore

Reply:
ok